lib/node_modules/@stdlib/assert/is-alphanumeric/docs/repl.txt

```diff
@@ -0,0 +1,28 @@
+
+{{alias}}( str )
+    Tests whether a string contains only alphanumeric characters.
+
+    Parameters
+    ----------
+    str: string
+        Input string
+
+    Returns
+    -------
+    bool: boolean
+        Boolean indicating whether a string contains only alphanumeric characters.
+
+    Examples
+    --------
+    > var bool = {{alias}}( ‘abc0123456789' )
```

Comment:
This will not run because of `‘` instead of `'` being used. Should change code to `var bool = {{alias}}( 'abc0123456789' )`

Reply:
Resolved!